hls4ml/backends/quartus/passes/recurrent_templates.py

```diff
@@ -92,6 +93,7 @@ def format(self, node):
         params['config_mult_h'] = f'config{node.index}_h_mult'
         params['act_t'] = '{}_config{}'.format(node.get_attr('activation'), str(node.index) + '_act')
         params['act_recurrent_t'] = '{}_config{}'.format(node.get_attr('recurrent_activation'), str(node.index) + '_rec_act')
+        params['pytorch'] = 'true' if "pytorch" in node.attributes.keys() else 'false'
```

Comment:
Very minor point: hls4ml seems to use single quotes more often (except for doc strings), so I would change "pytorch" to 'pytorch'. There are double quotes used in other places. But truthfully I am not convinced whether we need to change it and run the tests again, since this is so minor.